src/pratt.rs

```diff
@@ -513,16 +513,132 @@ macro_rules! impl_pratt_for_tuple {
 }
 
 impl_pratt_for_tuple!(
-    AA_ AB_ AC_ AD_ AE_ AF_ AG_ AH_ AI_ AJ_ AK_ AL_ AM_ AN_ AO_ AP_ AQ_ AR_ AS_ AT_ AU_ AV_ AW_ AX_ AY_ AZ_
-    BA_ BB_ BC_ BD_ BE_ BF_ BG_ BH_ BI_ BJ_ BK_ BL_ BM_ BN_ BO_ BP_ BQ_ BR_ BS_ BT_ BU_ BV_ BW_ BX_ BY_ BZ_
-    CA_ CB_ CC_ CD_ CE_ CF_ CG_ CH_ CI_ CJ_ CK_ CL_ CM_ CN_ CO_ CP_ CQ_ CR_ CS_ CT_ CU_ CV_ CW_ CX_ CY_ CZ_
-    DA_ DB_ DC_ DD_ DE_ DF_ DG_ DH_ DI_ DJ_ DK_ DL_ DM_ DN_ DO_ DP_ DQ_ DR_ DS_ DT_ DU_ DV_ DW_ DX_ DY_ DZ_
-    EA_ EB_ EC_ ED_ EE_ EF_ EG_ EH_ EI_ EJ_ EK_ EL_ EM_ EN_ EO_ EP_ EQ_ ER_ ES_ ET_ EU_ EV_ EW_ EX_ EY_ EZ_
-    FA_ FB_ FC_ FD_ FE_ FF_ FG_ FH_ FI_ FJ_ FK_ FL_ FM_ FN_ FO_ FP_ FQ_ FR_ FS_ FT_ FU_ FV_ FW_ FX_ FY_ FZ_
-    GA_ GB_ GC_ GD_ GE_ GF_ GG_ GH_ GI_ GJ_ GK_ GL_ GM_ GN_ GO_ GP_ GQ_ GR_ GS_ GT_ GU_ GV_ GW_ GX_ GY_ GZ_
-    HA_ HB_ HC_ HD_ HE_ HF_ HG_ HH_ HI_ HJ_ HK_ HL_ HM_ HN_ HO_ HP_ HQ_ HR_ HS_ HT_ HU_ HV_ HW_ HX_ HY_ HZ_
+    O1 O2 O3 O4 O5 O6 O7 O8
+    O9 O10 O11 O12 O13 O14 O15 O16
+    O17 O18 O19 O20 O21 O22 O23 O24
+    O25 O26
 );
 
+#[allow(unused_variables, non_snake_case)]
+impl<'a, Atom, T, const N: usize> Pratt<Atom, [T; N]> {
```

Comment:
I don't think this is doing what you think it's doing, unfortunately. This requires that all operators are of exactly the same time, which is almost useless. For this to be useful you'd need to implement `Operator` for `Box<dyn Operator>`, but then the code below like `T::IS_PREFIX` and others would need to change to check what kind of operator it is at run time.